manifests/plugin/certguard.pp

```diff
@@ -1,5 +1,6 @@
 # @summary Pulp Certguard plugin
 class pulpcore::plugin::certguard {
+  # TODO: this breaks - b79786c561ea56054c03fe83527cb79db84c295a should have addressed this
```

Comment:
@evgeni said on IRC that certguard in 3.15 depends on python3(8)-rhsm now so this should no longer be needed. The question is: do we still need to ensure it's installed on Pulpcore 3.14?